build/images/scripts/start_ovs_netdev

```diff
@@ -20,12 +20,24 @@ function fix_ovs_ctl {
     sed -i 's/\(\w*\)\(insert_mod_if_required || return 1\)/\1# \2/' /usr/share/openvswitch/scripts/ovs-ctl
 }
 
+# grep exits 1 if no matches are found. This function catches exit status 1 to
+# avoid failing if no matches are found. This still allows grep to return with
+# other exit status codes in the case of other failures.
+function grep_allow_no_match {
+  grep "$@" || test $? = 1;
+}
+
 # See http://docs.openvswitch.org/en/latest/howto/userspace-tunneling/
 function add_br_phy {
     log_info $CONTAINER_NAME "Creating OVS br-phy bridge for netdev datapath type"
     hwaddr=$(ip link show eth0 | grep link/ether | awk '{print $2}')
     inet=$(ip addr show eth0 | grep "inet " | awk '{ print $2 }')
+    inet6=$(ip addr show eth0 | grep_allow_no_match "inet6 " | grep_allow_no_match global | awk '{ print $2 }')
```

Comment:
nit: why is `"inet6"` quoted but not `global`?

Reply:
You are right. We were using `ip addr` so we needed to filter for the `inet6 ` address originally. Though we will switch to `ip -6 addr` and drop the `inet6 ` grep. Thanks!